tests/wpt/web-platform-tests/html/rendering/non-replaced-elements/tables/table-width.html

```diff
@@ -0,0 +1,25 @@
+<link rel="match" href="table-width-ref.html">
```

Comment:
This currently doesn't pass in Servo (for presumably unrelated reasons?) but it passes in Firefox and Chrome. If anyone has any insight into this or has a suggestion for a better test case for `width=0`, please let me know